### ISSUE TITLE
How to get compressed video dataset?

### ISSUE THREAD

**Issue by StevenTony611** [researchmm/FTVSR#8]
In the readme file, 'Generate the compressed videos by ffmpeg with command "ffmpeg -i LR.mp4 -vcodec libx264 -crf CRFvalue LR_compressed.mp4".' to obtain the compressed videos dataset, but REDS and Vimeo90K are the sequences datasets not *.mp4, how to get the compressed video dataset from these sequences datasets specifically?

**Comment by qiuzhongwei-USTB** [comment]
for generating compressed videos, you can compose videos from the sequence images by ffmpeg, and then use the compression command to compress videos, finally generating the compressed sequence images from the compressed videos by ffmpeg.

**Comment by StevenTony611** [comment]
Thank you very much for your reply and give us much useful help!  Could you provide the compressed sequence images of the two datasets we really need?  We are really worried that we will not be able to generate compressed data that is exactly the same as FTVSR and cause bad results. If you can provide the compressed sequence images of two datasets, we will gratitude you very much！

**Comment by qiuzhongwei-USTB** [comment]
Thank you very much for your interest in our work. 
First, I don't know if redistributing the dataset violates the author's policy, and I'll find out. 
Second, these two datasets are so large that I can't share the whole data set. If it is not against policy to distribute the dataset again, I will attempt to share the compressed test set.

**Comment by StevenTony611** [comment]
Thank you for your reply！ The two datasets are really so large and it is hard to upload to the platform.  If it is convenient for you, we can rely on the mail. We've already sent the email to your qiuzhongwei@xs.ustb.edu.cn. Thank you very much!

**Comment by StevenTony611** [comment]
Have another better way to release the compressed sequence images of training datasets?  
We really need these compressed datasets for our work. If you see the email earnestly please reply to us, thank you very much !!!

**Comment by qiuzhongwei-USTB** [comment]
> 

I think the better way of obtaining the dataset is to download it from the official website.
And then using ffmpeg to generate compressed videos.
